.github/workflows/ci.yml

```diff
@@ -28,11 +28,25 @@ jobs:
     container:
       image: bilelmoussaoui/flatpak-github-actions:gnome-41
       options: --privileged
+    strategy:
+      matrix:
+        arch: [x86_64, aarch64]
+      # Don't fail the whole workflow if one architecture fails
+      fail-fast: false
     steps:
-    - uses: actions/checkout@v2
-    - uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@v4
-      with:
-        bundle: telegrand.flatpak
-        manifest-path: build-aux/com.github.melix99.telegrand.Devel.json
-        run-tests: true
-        cache-key: flatpak-builder-${{ github.sha }}
+      - uses: actions/checkout@v2
+      # Docker is required by the docker/setup-qemu-action which enables emulation
+      - name: Install deps
+        run: dnf -y install docker
+      - name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: arm64
+      - uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@v4
+        with:
+          bundle: telegrand.flatpak
+          manifest-path: build-aux/com.github.melix99.telegrand.Devel.json
+          run-tests: true
+          cache-key: flatpak-builder-${{ github.sha }}
```

Comment:
This is very similar to the default value, so we can remove this.

Reply:
This seems to introduce the following errors when omitted:

x86_64
```
 Error: Failed to save cache: ReserveCacheError: Unable to reserve cache with key -x86_64, another job may be creating this cache.
``` 
aarch64
```
Error: Failed to save cache: ReserveCacheError: Unable to reserve cache with key -aarch64, another job may be creating this cache.
```

Reply:
Mh, for what I understand that's a bug of the action's current release (fixed in master), so... nevermind.

Reply:
Ok, If I understand correctly I am supposed to put it back in until v5 is released. Close this if I'm correct.